client/src/context/contact/ContactState.js

```diff
@@ -1,5 +1,5 @@
 import React, { useReducer } from "react";
-import uuid from "uuid";
```

Comment:
displays unnecessary errors https://stackoverflow.com/questions/60830848/attempted-import-error-uuid-does-not-contain-a-default-export-imported-as-u